pandas/io/formats/format.py

```diff
@@ -440,7 +440,6 @@ def _chk_truncate(self):
         max_rows = self.max_rows
 
         if max_cols == 0 or max_rows == 0:  # assume we are in the terminal
-                                            # (why else = 0)
```

Comment:
Hanging comment was causing a lint error. I figured it didn't add much so easier to remove than reword